### DISCUSSION TITLE
Cannot find module

### DISCUSSION THREAD

**Discussion by liydaco** [resend/react-email#1233] [Q&A] [answered]
### Describe the Bug

Can't import the email.jsx file from index

I'm using M1 mac

Tried using canary

### Link to the code that reproduces this issue

https://github.com/liydaco/mail-server

### To Reproduce

$npm i
$node index.js

### Expected Behavior

Expecting the Email file to render and postmark to send email:

removed options data and api key

  code: 'ERR_MODULE_NOT_FOUND'

### What's your node version? (if relevant)

v18.14.0

**Comment by lordelogos** [comment, marked as answer]
Hi @liydaco,

**Summary:**
The issue is with `node` and `jsx` files
Solution in this [repo](https://github.com/lordelogos/react-email-postmark).
- Clone the repo
- Update the api keys
- Run with `yarn dev`

## Explanation
Node js by default doesn't work with es6 imports and jsx files out of the box. There has to be some configuration to get them to work. By far the most popular way is to setup `Babel`.

### A bit on Babel
Babel is a powerful JavaScript compiler and when it comes to using JSX with ES6 imports in Node.js, Babel plays a significant role in transpiling JSX and handling ES6 modules.

Once we setup babel and the configuration, we can now transpile our code to a format that node can understand and run with no hassle.

I hope this helps you!

**Comment by vincenzodomina** [comment]
I solved it simply by adding `"jsx": "react-jsx"` to my `tsconfig.json` to let typescript compile jsx files
